examples/data-sources/snowflake_database/data-source.tf

```diff
@@ -0,0 +1,8 @@
+data "snowflake_database" "this" {
+    name = "DEMO_DB"
+}
+
+resource "snowflake_database" "backup" {
```

Comment:
I think you can remove this resource, you can just keep the data source as the example. Same for the one below